tensorflow/lite/python/lite.py

```diff
@@ -1349,7 +1349,7 @@ def _has_valid_tensors(self):
     Returns:
       Bool.
     """
-    return self._input_tensors and self._output_tensors
+    return self._input_tensors is not None and self._output_tensors is not None
```

Comment:
Can you leave it as-is for output tensor check? It's not consistent with commit message, and also seems weird to convert a model with zero outputs - if there are some use case for zero outputs, please let me know.

I can think of a model with side effect in custom op, but not sure if it's actually useful

Reply:
Sorry for the late reply. There is indeed a bit inconsistent with the commit message，but as you said there maybe has no outputs if the final node is just a `print` op or other custom op with side effect. I am not sure whether we should take these cases into consideration or not, and whether the conversion tool would apply DCE optimization or not.

If it is determined that the zero outputs needs to be considered, I will correct the commit message, otherwise leave it as-is for output tensor check. So what do you think? Thanks.

Reply:
I think it would be safer to only modify input tensor check. @abattery, can you comment?

Reply:
@teijeong Hi teijeong, I have modified the codes. Take a look please, thanks.

Reply:
Could you add test cases into lite_test.py and lite_v2_test.py for this new behavior?

Reply:
@hjchen2 Can you please check @abattery's comments and keep us posted ? Thanks!